### PR TITLE
Creating items

### PR DESCRIPTION
**Pull request by andllrew**
**Ticket:** [Creating Items](https://cs375-pos-group-project.atlassian.net/jira/core/projects/ATS/board?selectedIssue=ATS-12)

**PR Desc:**

- Added a POST API endpoint to add items to the database `item` in server.js.
- Created an HTML page that owners will be directed to when they want to create an item.
- Created a local js file that will be multifunctional across different HTML pages.
- Added a fetch function that displays categories from the database `item_category`.
- Added a fetch function that sends item data to the backend POST body to add into the database `item`.

### CODE REVIEW

File: app/public/pos/create.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        },
+        body: JSON.stringify(item),
+    })
+    .then(response => {
+        if (!response.ok) {
+            message.textContent = "Failure";
+        }
+        else {
+            message.textContent = "Success";
+        }
```
Comment (gitnamehere) [inline app/public/pos/create.js:51]: You can use a ternary to shorten this down
```suggestion
        message.textContent = (response.ok ? "Success" : "Failure");
```

File: app/public/pos/create.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    })
+    .catch(error => {
+        console.log(error);
+    })
+});
```
Comment (gitnamehere) [inline app/public/pos/create.js:56]: add a new line here (a nitpick, but usually you want to end files with an empty line)

File: app/public/pos/create.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	</div>
+	<button id="submit">Create Item</button>
+</div>
+
+<div id="message"></div>
+
+<script src="create.js"></script>
+
+</body>
+</html>
```
Comment (gitnamehere) [inline app/public/pos/create.html:40]: nitpick: newline

**Review by gitnamehere — CHANGES_REQUESTED** [review]
nice work, just have a few suggested changes here

File: app/public/pos/create.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+<div class="item">
+	<div>
+		<label for="category">Category:</label>
+		<select id="category" name="category">
+		</select>
+	</div>
+	<div>
+		<label for="name">Name:</label>
+		<input id="name" type="text" name="name", size=40>
```
Comment (gitnamehere) [inline app/public/pos/create.html:22]:
```suggestion
		<input id="name" type="text" name="name" maxlength="50">
```

**Comment by wetefok** [comment]
I think it'd be best to move everything into a pos/item-management/ directory for better organization.

File: app/server.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 const express = require("express");
 const { Pool } = require("pg");
 const env = require("../env.json");
+const { error } = require("console");
```
Comment (gitnamehere) [inline app/server.js:4]: I think your vscode might've accidentally imported this

**Review by gitnamehere — APPROVED** [review]
FIRE CHANGES 🔥 LGTM

**Review by nhokzminh698 — APPROVED** [review]
ate the code up!!

**Pull request merged**